build/ci/evergreen.yml

```diff
@@ -2064,24 +2150,42 @@ buildvariants:
       - name: pkg_test_docker_ubuntu2004
       - name: pkg_test_docker_debian9
       - name: pkg_test_docker_debian10
-  - name: release_msi
-    display_name: "Release (go-msi)"
+  - name: release_mongocli_msi
+    display_name: "Release MongoCLI (go-msi)"
     run_on:
       - windows-vsCurrent-small
     expansions:
       go_root: "c:\\golang\\go1.17"
       go_bin: "c:\\golang\\go1.17/bin"
     tasks:
-      - name: release_msi
+      - name: release_mongocli_msi
   - name: release_github
-    display_name: "Release (goreleaser)"
+    display_name: "Release MongoCLI (goreleaser)"
+    run_on:
+      - macos-1014-codesign
+    expansions:
+      go_root: "/opt/golang/go1.17"
+      go_bin: "/opt/golang/go1.17/bin"
+    tasks:
+      - name: release_mongocli
+  - name: atlascli_release_github
```

Comment:
I will rename this variant to `release_atlascli_github`  and update the regex in the mongocli evergreen project to select the correct variants once we are ready to release atlascli. Currently, all the variants that starts with `release_` are triggered when a new tag is created

Reply:
let's add a TODO and a CLOUDP with this comment

Reply:
https://jira.mongodb.org/browse/CLOUDP-113305